### DISCUSSION TITLE
OpenSSL::PKey::RSAError Exception: EVP_PKEY_decrypt: padding check failed

### DISCUSSION THREAD

**Discussion by sushmakumaran** [openssl/openssl#25622] [Q&A]
I'm facing this issue after the ruby upgrade to 3.2.0. failing at: params.try(:decrypt, Rails.configuration.strongbox.passphrase).to_s
Openssl version used : OpenSSL 1.0.2u

**Comment by t8m** [comment]
OpenSSL 1.0.2u is out of support for a very long time.
